content/documentation/features/index.adoc

```diff
@@ -22,25 +22,25 @@ menu:
 
 == Overview
 
-Kiali helps you define, validate, and observe your Istio service mesh. Kiali provides answers to the questions: What are the microservices in my service mesh, and how are they connected?
+Kiali helps you define, validate, and observe the connections of your microservices in an Istio service mesh.
 
-Kiali works with Istio in OpenShift or Kubernetes. It visualizes the service mesh topology and provides visibility into features like request routing, circuit breakers, request rates, latency and more. Kial offers insights about the mesh components at different levels, from abstract Applications to Services and Workloads.
+Kiali works with Istio in OpenShift or Kubernetes. It visualizes the service mesh topology and provides visibility into features like request routing, circuit breakers, request rates, latency and more. Kiali offers insights about the mesh components at different levels, from abstract applications to services and workloads.
 
-icon:lightbulb[size=2x] {nbsp}{nbsp}{nbsp}{nbsp} Kiali also includes Jaeger Tracing to provide distributed tracing out of the box.
+icon:lightbulb[size=2x] {nbsp}{nbsp}{nbsp}{nbsp} Kiali also includes Jaeger Tracing to provide distributed tracing by default.
 
 toc::[]
 
 == Observability Features
 
-Observability is being able to see your service mesh in operation.  It combines topology, telemetry, traces, logs, events and definitions. It help you ensure your mesh is healthy or to quickly identify problem areas.  It helps correlate different information sources into a holistic view of the system.
+The following obeservability features help you ensure your mesh is healthy or to quickly identify problem areas in operation. It combines topology, telemetry, traces, logs, events and definitions in a holistic view of your system.
 
 
 === Graph
-The graph provides a powerful way to visualize the topology of your service mesh.  It shows you which services communicate with each other and the **traffic rates** and **latencies** between them.  It lets you visually identify problem areas ane quickly pinpoint issues.  Kiali provides **four different graph types** giving you a high-level view of service interactions, a low level view of pods, or a logical view of applications.
+The graph provides a powerful way to visualize the topology of your service mesh. It shows you which services communicate with each other and the traffic rates and latencies between them, which helps you visually identify problem areas ane quickly pinpoint issues. Kiali provides graphs that show a high-level view of service interactions, a low level view of pods, or a logical view of applications.
```

Comment:
typo: ane -> any

also, a change outside of the edits:

a low level view of pods ->  a low level view of workloads